src/opm/parser/eclipse/share/keywords/000_Eclipse100/B/BDENSITY

```diff
@@ -0,0 +1,3 @@
+{"name" : "BDENSITY", "sections" : ["PROPS"], "size" : {"keyword":"TABDIMS", "item":"NTPVT" }, "records" :
```

Comment:
The `records: [[ ]]` should be used for the keywords where the records have different structure, that is not the case here and you should use `items:`
```json
..."items" : [{"name" : "BRINE_DENSITY"   , "value_type" : "DOUBLE" ,"size_type" : "ALL"  , "dimension" : "Density"}]
```